<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseItem</class>
 <widget class="QDialog" name="ExpenseItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>232</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>ExpenseItem</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="LocalTransportation">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>市内交通费</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="LodgingExpenses">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>实习费</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="TrainingFees">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>培训费</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="OfficeSupplies">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>办公用品</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="StudentActivityFee">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学生活动费</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="EntertainmentExpense">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>招待费</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="PurchaseOfProfessionalMaterials">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>专业材料购置费</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="Correspondence">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通讯费</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="ConstructionOfFixedAssets">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>固定资产构建费</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="PurchaseOfExperimentalMaterials">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>实验材料购置费</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="PostFee">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>邮寄费</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="Streamer">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>横幅</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="SportingGoodsPurchaseFee">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>体育用品购置费</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="SelfServiceCost">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>自备车费用</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="Other">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>其他</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="XeroxFee">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>资料复印费</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="MotorVehicleMaintenance">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>机动车维修费</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="Typography">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>版面印刷费</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QRadioButton" name="OfficeEquipmentMaintenance">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>办公设备维修费</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="MaterialTestingAndProcessingFee">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>材料测试加工费</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QRadioButton" name="EquipmentMaintenanceFee">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>仪器设备维修费</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
